parsers/EIA.py

```diff
@@ -61,6 +130,10 @@ def _fetch_production_or_consumption(zone_key, series_id, session=None,
         raw_data = series.last(24)
 
     # UTC timestamp with no offset returned.
+    if not raw_data.get('series'):
+        # Series doesn't exist. Probably requesting a fuel from a region that
+        # doesn't have any capacity for that fuel type.
+        return []
```

Comment:
Maybe we could log a warning here, the empty list might cause some odd looking errors in the functions that are calling `_fetch_series`

Reply:
yeah, i considered this. this is normal for fuels that aren't used in a given region, though, so we'd log a bunch of these warnings every time the parser runs, which seemed like the wrong idea.

i think all of the `_fetch_series()` calls in this module should handle this empty list fine. i confirmed that `merge_production_outputs()` handles it ok, and the rest at most just iterate over the returned list.

Reply:
Yup ok for the fuel types. Maybe we could do something when the iteration fails?